include/WCSimWCDigi.hh

```diff
@@ -195,29 +201,32 @@ public:
 
     for (i = 1; i < (int) time.size(); ++i)
       {
-        index_time  = time.at(i);
-        index_timepresmear  = time_presmear.at(i);
-        index_pe = pe.at(i);
-        if(sort_digi_compositions) index_digicomp = fDigiComp.at(i);
-        index_trackID = trackID.at(i);
-        index_parentSavedTrackID = parentSavedTrackID.at(i);
-	index_photonstarttime = photonStartTime[i];
-	index_photonstartpos = photonStartPos[i];
-	index_photonendpos = photonEndPos[i];
-	index_photonstartdir = photonStartDir[i];
-	index_photonenddir = photonEndDir[i];
+        index_time         = time.at(i);
+        index_timepresmear = time_presmear.at(i);
+        index_pe           = pe.at(i);
+        if(sort_digi_compositions) 
+        index_digicomp             = fDigiComp.at(i);
```

Comment:
hmm I mean use a single line `if(sort_digi_compositions) index_digicomp = fDigiComp.at(i);` as in the original code

Reply:
👍